cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/internal/query/IndexTest.java

```diff
@@ -126,13 +127,13 @@ public void comparesIndexSettingsAndReturnsEquality() {
     @Test
     public void returnsIndexSettingsAsAString() {
         Index index = new Index(fieldNames, indexName, IndexType.TEXT);
-        assertThat(IndexCreator.settingsAsJSON(index), is("{\"tokenize\":\"simple\"}"));
+        assertThat(CreateIndexCallable.settingsAsJSON(index), is("{\"tokenize\":\"simple\"}"));
```

Comment:
With respect to my other comment about keeping `settingsAsJSON` `protected`, we could move these two tests out to their own class in the `query.callables` package to achieve this.